data/json/itemgroups/magazines.json

```diff
@@ -141,7 +141,6 @@
       ["calicomag", 15],
       ["fnp90mag", 15],
       ["mp5bigmag", 10],
-      ["ppsh41mag", 10 ],
```

Comment:
Ah right I see. Those are both obscure magazines though and so presumably better belong in that category?